packages/opentelemetry-core/test/common/attributes.test.ts

```diff
@@ -102,5 +117,78 @@ describe('attributes', () => {
       assert.ok(Array.isArray(attributes.arr));
       assert.strictEqual(attributes.arr[0], 'unmodified');
     });
+
+    describe('value sanitizers', () => {
+      describe('http.url', () => {
+        it('should remove username and password from the url', () => {
+          const out = sanitizeAttribute(
+            SemanticAttributes.HTTP_URL,
```

Comment:
Would you mind converting those test cases into table-based tests to reduce the repetition between tests?

```js
const testData = [
  {
    name: 'should remove username and password from the url',
    input: 'http://user:pass@host:9000/path?query#fragment', 
    expect: 'http://host:9000/path?query#fragment',
  },
  ...
];

for (const item of testData) {
  it(item.name, () => {
    ...
  });
}
```

Reply:
sure that would be possible.